scholarx/past-programs/index.html

```diff
@@ -48,7 +48,7 @@
           href="style.css">
 </head>
 
-<body style="font-family: 'Montserrat', sans-serif; color: black">
+<body style="font-family: 'Montserrat', sans-serif;">
```

Comment:
Let's create a CSS class for this and add this to the global CSS

Reply:
So I should remove the inline css right? @anjula-sack

Reply:
Done can you review this now @anjula-sack ?